hoodie-cli/src/main/java/com/uber/hoodie/cli/commands/CommitsCommand.java

```diff
@@ -131,8 +132,13 @@ public String rollbackCommit(@CliOption(key = {"commit"}, help = "Commit to roll
     }
 
     SparkLauncher sparkLauncher = SparkUtil.initLauncher(sparkPropertiesPath);
-    sparkLauncher
-        .addAppArgs(SparkMain.SparkCommand.ROLLBACK.toString(), commitTime, HoodieCLI.tableMetadata.getBasePath());
+
+    SparkMain.HoodieRollbackCommandConfig config = new SparkMain.HoodieRollbackCommandConfig();
```

Comment:
To keep things consistent, Can you define a separate main class HudiRollback similar to com.uber.hoodie.utilities.HoodieCompactor and define both config and move the rollback  calling logic from SparkMain to the new class ?

Rollback to savepoint should be handled in the same new class  com.uber.hoodie.utilities.HudiRollback